data/campaigns/Sceptre_of_Fire/scenarios/9_Caverns_of_Flame.cfg

```diff
@@ -434,430 +1027,50 @@
-        [/message]
-        [objectives]
+        [filter_condition]
+            [variable]
+                name=turn_number
+                less_than=6
+            [/variable]
+        [/filter_condition]
+        {MOVE_UNIT (id=Gathor) 36 5}
+        [remove_shroud]
             side=1
-            [objective]
-                description= _ "Move onto the glyph in the center of the volcano"
-                condition=win
-            [/objective]
-            [objective]
-                description= _ "Death of Rugnur"
-                condition=lose
-            [/objective]
-            [objective]
-                description= _ "Death of Krawg"
-                condition=lose
-            [/objective]
-            [objective]
-                description= _ "Death of Thursagan"
-                condition=lose
-            [/objective]
-
-            {IS_LAST_SCENARIO}
-        [/objectives]
-    [/event]
-
-#define CHANGE_MAP TURN
-    [if]
-        [variable]
-            name=overlayturn
-            numerical_equals={TURN}
-        [/variable]
-        [then]
-            [terrain_mask]
-                x,y=1,1
-                border=yes
-                mask="{campaigns/Sceptre_of_Fire/maps/9_overlay/9_o{TURN}.mask}"
-                [rule]
-                    old=Ql,Uu*,C*,*^V*,R*,Tb*,Uh*,Ur*
-                    new=Ql
-                    terrain=Qlf
-                [/rule]
-                [rule]
-                    use_old=yes
-                [/rule]
-            [/terrain_mask]
-            [redraw][/redraw]
-        [/then]
-    [/if]
-#enddef
-
-    #triggering the volcano
-    [event]
-        name=moveto
-        [filter]
-            x,y=18,23
+            x,y=32-43,3-9
+        [/remove_shroud]
+        [redraw]
             side=1
-        [/filter]
-        {QUAKE "rumble.ogg"}
-        {QUAKE "rumble.ogg"}
-        [message]
-            speaker=Thursagan
-            message= _ "That’s it!  You've triggered the volcanic eruption!"
-        [/message]
-        [terrain]
-            x=11
-            y=4-5
-            terrain=Uu
-        [/terrain]
-        [terrain]
-            x=20
-            y=3-4
-            terrain=Uu
-        [/terrain]
-        {QUAKE "rumble.ogg"}
-        {QUAKE "rumble.ogg"}
-        [message]
-            speaker=Rugnur
-            message= _ "Look, some walls have collapsed!  Those orcs were trapped, but now they've a way out!"
-        [/message]
-        [scroll_to]
-            x,y=14,1
-        [/scroll_to]
-        {QUAKE "rumble.ogg"}
-        {QUAKE "rumble.ogg"}
+        [/redraw]
         [message]
-            speaker=Baglur
-            message= _ "We’ve succeeded in stopping the elves from getting the Sceptre, I'd say.  But what do we do now?!"
+            speaker=Gathor
+            message= _ "Ah ha!  I knew it, there are intruders!  And yet our damned bridge is broken, it was built by idiots.  Come, come, bring planks, we've got to get across!"
```

Comment:
The double-space should be a single-space.

Reply:
fixed